examples/common/maths.ads

```diff
@@ -1,11 +1,32 @@
-with GL.Types;
+
+with GL.Types; use GL.Types;
 
 package Maths is
+
+    subtype tDegree is Single;
+    subtype tRadian is Single;
+
+    function Degree (Radians : tRadian) return tDegree;
     procedure Init_Lookat_Transform
-      (Position, Target, Up : GL.Types.Singles.Vector3;
-       Look_At              : out GL.Types.Singles.Matrix4);
+      (Position, Target, Up : Singles.Vector3; Look_At : out Singles.Matrix4);
+    procedure Init_Orthographic_Transform (Bottom, Top, Left, Right,
+                                           Z_Near, Z_Far : Single;
+                                           Transform     : out Singles.Matrix4);
     procedure Init_Perspective_Transform
-      (View_Angle, Width, Height, Z_Near, Z_Far : GL.Types.Single;
-       Transform                                : out GL.Types.Singles.Matrix4);
+      (View_Angle, Width, Height, Z_Near, Z_Far : Single;
+       Transform                                : out Singles.Matrix4);
+    function Length (V : Singles.Vector3) return Single;
+    procedure Normalize (V : in out Singles.Vector3);
```

Comment:
Since normalization is a mathematical function, I would prefer this to be a function named `Normalized` which returns a new `Vector3`. That is more versatile as it can be used for variable initialization and such.